### ISSUE TITLE
Theme for iTerm please

### ISSUE THREAD

**Issue by srour** [LegoYoda112/nostromo_ui_themes#8]
This is great. Can we get one for iTerm please? :)

**Comment by dhrm1k** [comment]
Yes, i am in middle of exams. I just made a similar theme for vim. I will get on it.

**Comment by rwinslow** [comment]
i just made an iterm theme with nostromo colors. @LegoYoda112 let me know if you want me to send it to you or have you grant me the permissions to create a PR

**Comment by pedrodim** [comment]
Checkout #14
